deltachat-ios/en.lproj/Localizable.strings

```diff
@@ -894,3 +894,6 @@
 
 "webxdcs" = "Apps";
 "webxdc_empty_hint" = "Webxdc apps you have received or sent will appear here. You can tap on files to select downloaded apps.";
+"shortcut_share_btn" = "Click the share button.";
+"shortcut_add_to_home_description" = "Select - To home screen - to add the shortcut to your home screen!";
```

Comment:
```suggestion
"shortcut_share_btn" = "Click the share button";
"shortcut_add_to_home_description" = "Select \"Add to Home Screen\" to add the app to your home screen";
```

this should also better go to `untranslated.xml`, or?

Reply:
right

Reply:
Escaped quotes screw the leayout and I couldn't yet figure out how to properly escape them. `&quot;` is just printed literally, native JS encoding `\x22` didn't work either out of the box. I appreciate any hints how to fix that. For now I reverted the escaped quotes and use again dashes (that was also the reason I used them in first place)